### DISCUSSION TITLE
Thoughts on making the editor open source as well

### DISCUSSION THREAD

**Discussion by lydamian** [thomaspoignant/go-feature-flag#1200] [Q&A] [answered]
Thoughts on making the editor open source as well, possibly providing a docker image so that it can be self hosted

**Comment by thomaspoignant** [comment, marked as answer]
@lydamian sorry about the delay in my response I haven't checked the discussions recently.

The editor is already Open source and is available on the website, you can access the source directly here: https://github.com/thomaspoignant/go-feature-flag/tree/main/website/src/pages/editor
